### DISCUSSION TITLE
Update Logs

### DISCUSSION THREAD

**Discussion by Dvalin21** [manuc66/node-hp-scan-to#840] [General]
Hello,

Just received a notification for image update on this docker. However, I'm probably looking over where any change logs are. Anyway, thanks for all the work in this. Its simple yet amazing app. Love using it with paperless-ngx docker app.

**Comment by manuc66** [comment]
Hello @Dvalin21

I do try to [maintain a changelog and I do emit some versionning](https://github.com/manuc66/node-hp-scan-to/releases) but those are only for the npm command line not for the docker image as for now.

At the momment the docker image is updated each time there is a nex commit on the main branch

**Reply by Dvalin21** [reply]
Thanks for the update!
